lib/Core/Executor.h

```diff
@@ -322,6 +322,11 @@ class Executor : public Interpreter {
 
   void executeMarkPersistent(ExecutionState &state, ref<ConstantExpr> address);
   void executeMarkPersistent(ExecutionState &state, const MemoryObject *mo);
+  bool isPersistent(ExecutionState &state, const MemoryObject *mo);
```

Comment:
(nit) Maybe rename to "isPersistentMemory" because "isPersistent" looks a lot like "isPersisted" on first parse.

Reply:
Agreed, looks like `isPersisted`.